tensorboard/webapp/feature_flag/store/feature_flag_selectors_test.ts

```diff
@@ -415,4 +415,29 @@ describe('feature_flag_selectors', () => {
       expect(selectors.getIsDataTableEnabled(state)).toEqual(true);
     });
   });
+
+  describe('#getIsLinkedTimeProspectiveFobEnabled', () => {
+    it('returns the proper value', () => {
+      let state = buildState(
+        buildFeatureFlagState({
+          defaultFlags: buildFeatureFlag({
+            enabledScalarDataTable: false,
+          }),
+        })
+      );
+      expect(selectors.getIsDataTableEnabled(state)).toEqual(false);
+
+      state = buildState(
+        buildFeatureFlagState({
+          defaultFlags: buildFeatureFlag({
+            enabledScalarDataTable: false,
+          }),
+          flagOverrides: {
+            enabledScalarDataTable: true,
+          },
+        })
+      );
+      expect(selectors.getIsDataTableEnabled(state)).toEqual(true);
```

Comment:
I imagine this is just a copy paste situation and you were intending to test the new feature flag?
```suggestion
      let state = buildState(
        buildFeatureFlagState({
          defaultFlags: buildFeatureFlag({
            enabledProspectiveFob: false,
          }),
        })
      );
      expect(selectors.getIsLinkedTimeProspectiveFobEnabled(state)).toEqual(false);

      state = buildState(
        buildFeatureFlagState({
          defaultFlags: buildFeatureFlag({
            enabledProspectiveFob: false,
          }),
          flagOverrides: {
            enabledProspectiveFob: true,
          },
        })
      );
      expect(selectors.getIsLinkedTimeProspectiveFobEnabled(state)).toEqual(true);
```

Reply:
Ah so embarrassing missed the edit here. Thanks Riley